mask_to_polygons/processing/score.py

```diff
@@ -31,7 +31,17 @@ def spacenet(predictions, ground_truth):
 
     tree = shapely.strtree.STRtree(ground_truth)
 
+    def make_valid(p):
+        if p.is_valid:
+            return p
+        else:
+            (minx, miny, maxx, maxy) = p.bounds
+            stretch = max(maxx - minx, maxy - miny)
+            return p.buffer(stretch * 0.01)
```

Comment:
What's the rationale for the stretch? In other parts of RV we just do p.buffer(0) which seems to work.

Reply:
If that addresses bow ties then it is probably fine.

Reply:
Looks like using a small constant like you did is mentioned here (and might work better). I would just add a comment with the link to this issue since it might appear odd to someone not familiar with the bowtie problem. https://github.com/Toblerity/Shapely/issues/462